scene/3d/path_3d.cpp

```diff
@@ -191,6 +191,9 @@ void PathFollow3D::_update_transform(bool p_update_xyz_rot) {
 		t.origin = pos;
 	} else {
 		t = c->sample_baked_with_rotation(progress, cubic, false);
+		if (z_forward) {
+			t.basis = Basis(t.basis.get_column(1).normalized(), Math_PI) * t.basis;
```

Comment:
Performing this conversion by using the axis-angle Basis constructor is not the ideal solution. I think this code will perform an equivalent operation:

```suggestion
			t.basis *= Basis::from_scale(-1.0, 1.0, -1.0);
```

(Linear algebra trick: A negative scale on two axes performs a 180 degree rotation on the plane formed by those axes. And we are doing this from the right so it will be in local space.)